Makefile

```diff
@@ -143,9 +143,9 @@ endif
 
 cmd/agentctl/agentctl: cmd/agentctl/main.go
 ifeq ($(CROSS_BUILD),false)
-	CGO_ENABLED=1 go build $(GO_FLAGS) -o $@ ./$(@D)
+	CGO_ENABLED=0 go build $(GO_FLAGS) -o $@ ./$(@D)
 else
-	@CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM); $(seego) build $(CGO_FLAGS) -o $@ ./$(@D)
+	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM) go build $(GO_FLAGS) -o $@ ./$(@D)
```

Comment:
@56quarters heads up, found this in release testing. Adding config validation imported promtail from agentctl, which accidentally added a dependency on libsystemd-dev being installed, and was causing `make agentctl-image` to fail locally. The CI builds a cross-platform image so it wasn't caught there. 

agentctl doesn't need CGO, so I've just disabled it here to fix the problem.

Reply:
Gah, sorry about that.

Reply:
No worries!